src/daemon/utils_ssl.c

```diff
@@ -0,0 +1,100 @@
+/**
+ * collectd - src/utils_ssl.c
+ * Copyright (C) 2015  Toni Moreno
+ *
+ * This program is free software; you can redistribute it and/or modify it
```

Comment:
An an example code I think we can use the licensing that you prefer. But I'm not an expert on open licences , so I prefer delegate this question on you and select the licence that you think is better for collectd.
What do you thing ? Could you give me a hint on this?